File: Core/Entities/Comment.cs
﻿using System.ComponentModel.DataAnnotations.Schema;

namespace EmployeeRecognition.Core.Entities;

[Table("comments")]
public class Comment
{
    public int Id { get; set; }
    public required int KudoId { get; set; }
    public required string SenderId { get; set; }
    public required string SenderName { get; set; }
    public required string SenderAvatar { get; set; }
    public required string Message { get; set; }

}

File: Web/ClientApp/src/app/interfaces/comment.ts
export interface CommentDto {
    kudoId: number;
    senderId: string;
    message: string;
}
  
export interface Comment {
    id: number;
    kudoId: number;
    senderId: string;
    senderName: string;
    senderAvatar: string;
    message: string;
}